docs/pages/concepts/offloading/resource-reflection.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Resource reflection
+weight: 4
+---
+
+In addition to Node and Pods lifecycle handling, the Liqo virtual kubelet implements a feature we call *resource reflection*.
+It is responsible for the propagation and synchronization of selected control plane information into remote clusters, to enable the seamless execution of offloaded Pods. Currently, the virtual kubelet supports the reflection of the following Kubernetes resources:
+
+* `Services`, abstracting the access to workloads spread across multiple clusters, while enabling the usage of standard DNS discovery mechanisms.
+* `EndpointSlices`, synchronizing the endpoints associated with the reflected services, regardless of whether the corresponding Pods are running in the same or in a different cluster.
+* `ConfigMaps`, typically holding configuration data consumed by Pods.
+* `Secrets`, typically holding secret data consumed by Pods.
+
+Once a given namespace is enabled for [Liqo extension]((/usage/namespace_offloading)) (i.e., through the creation of a `NamespaceOffloading` resource, either manually or through the appropriate `liqoctl` command), the virtual kubelet starts reflecting all resources belonging to the above categories into the subset of selected clusters (i.e., by means of the `ClusterSelector` field).
+The local copy of each resource is the source of trust leveraged to realign the content of the *shadow* copy reflected in each remote cluster.
+Yet, appropriate remapping of certain information (e.g., [endpoints IP](http://localhost:1313/concepts/networking/components/network-manager/#reflection)) is transparently performed by the virtual kubelet, accounting for conflicts and different configurations in different clusters.
+
+### Service and EndpointSlice reflection
+
+The reflection of Service and EndpointSlice resources is a key element to allow the seamless intercommunication between microservices spread across multiple clusters.
+
+As a matter of example, let consider the scenario shown in figure below, depicting an application composed of three Pods (partially hosted by a local cluster and partially offloaded to a remote one through a virtual node) and exposed through their respective Service (then, Kubernetes automatically creates the corresponding EndpointSlice).
```

Comment:
```suggestion
As a matter of example, let consider the scenario shown in the figure below, depicting an application composed of three Pods (partially hosted by a local cluster and partially offloaded to a remote one through a virtual node) and exposed through their respective Service (then, Kubernetes automatically creates the corresponding EndpointSlice).
```
(missing the figure, though :-( )

Reply:
I know, that is the main reason why this PR was still in pending :)